### DISCUSSION TITLE
Why can't I select a specific private repository?

### DISCUSSION THREAD

**Discussion by sachinsenal0x64** [community/community#113430] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/127573781/418c1e88-ad26-4557-ae55-48e462f25356)

**Comment by santoshvandari** [comment, marked as answer]
It is clearly stated that the repository must be hosted by an Organizational account and You need to be an Admin of that account. 
If you are already, then you can do it by clearing cache once.

**Reply by sachinsenal0x64** [reply]
Thanks :)
